### DISCUSSION TITLE
[FR] How to compile it on windows?

### DISCUSSION THREAD

**Discussion by Nagisakurumi** [LSPosed/LSPosed#946] [Q&A] [answered]
**Detail**:
I have an error  "Could not determine the dependencies of task ':app:compileReleaseJavaWithJavac'." !!!
it's import error when it import aidl in android studio.

**Comment by yujincheng08** [comment, marked as answer]
#943

**Comment by Nagisakurumi** [comment]
I uses cmd "gradlew zipRelease zipDebug"  then it has error 


* What went wrong:
Could not determine the dependencies of task ':app:mergeReleaseNativeLibs'.
> Could not resolve all task dependencies for configuration ':app:releaseRuntimeClasspath'.
   > Could not resolve project :interface.
     Required by:
         project :app > project :manager-service
      > No matching configuration of project :interface was found. The consumer was configured to find a runtime of a component, preferably optimized for Android, as well as attribute 'com.and
roid.build.api.attributes.BuildTypeAttr' with value 'release' but:
          - None of the consumable configurations have attributes.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

**Reply by viki3d** [reply]
**git submodule update --init --recursive** 
will fix this problem

**Comment by yujincheng08** [comment]
google git submodule.
